cloudbuild.yaml

```diff
@@ -0,0 +1,16 @@
+steps:
+  - id: go_version
+    name: 'gcr.io/cloud-builders/go'
+    args: ['version']
+    env: ['GOPATH=/gopath', 'GO111MODULE=on']
+  - id: go_build
+    name: 'gcr.io/cloud-builders/go'
+    args: ['test', './...']
+    env: ['GOPATH=/gopath', 'GO111MODULE=on']
+  - id: go_cover
+    name: 'gcr.io/cloud-builders/go'
+    env: ['GOPATH=/usr/local/go', 'GO111MODULE=on', 'GOFLAGS=-mod=vendor']
```

Comment:
I don't know much about cloudbuild.yaml files, so maybe I shouldn't be reviewing this, but why does this step have a different GOPATH to the others?